tests/library/test_metrics.py

```diff
@@ -388,6 +471,115 @@ def test_token_overlap(self):
         for target, value in global_targets.items():
             self.assertAlmostEqual(value, outputs[0]["score"]["global"][target])
 
+    def test_grouped_instance_metrics(self):
```

Comment:
if these are the same tests as in [grouped_instance_metrics.py](https://github.com/IBM/unitxt/pull/452/files?file-filters%5B%5D=.py&show-viewed-files=true#diff-a3f7cbec6d34ae3240ec3ad5e6198dc6f2ccc1ab9edb5fb47bcda4595e246285), then its not needed to do this twice... the grouped_instance_metrics.py runs as part of the tests, so its ok to assume that repeating what it does is not needed..

Reply:
they are the same tests, right?

Reply:
Elron said this is ok